packages/mobile/package.json

```diff
@@ -37,9 +37,9 @@
     "nodejs-mobile-react-native": "^0.8.0",
     "pkijs": "*",
     "pvutils": "^1.0.17",
-    "react": "17.0.1",
+    "react": "18.0.0",
```

Comment:
Can we confirm this is safe for the rest of the project?

Reply:
hmm its package.json in mobilce, should not have a impact on the rest of the project